code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/MessagingFullscreenMessageDelegate.java

```diff
@@ -55,6 +57,10 @@ public void onShow(final FullscreenMessage fullscreenMessage) {
      */
     @Override
     public void onDismiss(final FullscreenMessage fullscreenMessage) {
```

Comment:
i believe this change to onDismiss is not needed. dismiss tracking will be triggered at https://github.com/adobe/aepsdk-messaging-android/blob/main/code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/MessagingFullscreenMessageDelegate.java#L146.

Reply:
On testing, it didn't get tracked in that method because we pass true for `supressAutoTrack` variable.

I compared with iOS which does the same by calling https://github.com/adobe/aepsdk-messaging-ios/blob/e18c48a14c461a5be68c2168b0bd3df2e20b44b9/AEPMessaging/Sources/Message+FullscreenMessageDelegate.swift#LL38C17-L38C24